test/CadlRanchProjects/type/model/inheritance/not-discriminated/src/Generated/Models/Cat.cs

```diff
@@ -17,7 +17,7 @@ public partial class Cat : Pet
         /// <param name="name"></param>
         /// <param name="age"></param>
         /// <exception cref="ArgumentNullException"> <paramref name="name"/> is null. </exception>
-        internal Cat(string name, int age) : base(name)
+        public Cat(string name, int age) : base(name)
```

Comment:
looks like that we have this change because the usage is bubbled up from the derived type to this base.
From the property's perspective, this is correct.
But I wonder if we have a way to keep this ctor internal. There is a bit difference between when the model itself has a input usage, and when the model gets a input usage from its derived type.